### DISCUSSION TITLE
BitCanna Relayer

### DISCUSSION THREAD

**Discussion by ewais001** [archway-network/discussions#15] [Relayers]
We'd love an IBC channel to be spun up between Archway and BitCanna.

**Comment by loverdos** [comment]
> We'd love an IBC channel to be spun up between Archway and BitCanna.

Could you please provide more context and details? 

We recently published some relevant [guidelines](https://github.com/orgs/archway-network/discussions/13).

**Reply by ewais001** [reply]
We'd like to list the BCNA token on Astrovault. That requires the creation and maintenance of a BitCanna-Archway IBC channel and relayer. 

BitCanna also created Cosmos Pay, a dope widget which could hopefully use ARCH for payments eventually.
